conjur-openapi.yml

```
@@ -25,6 +25,8 @@ tags:
   description: "SSH keys"
 - name: "resources"
   description: "Resources"
+- name: "status"
+  description: "Status of Conjur"
```

```suggestion
  description: "Server status"
```

case should be consistent with examples above